testing/tests/simple.rs

```diff
@@ -72,6 +72,36 @@ fn test_variables_no_escape() {
     );
 }
 
+const FOO: &str = "FOO";
+const FOO_BAR: &str = "FOO BAR";
```

Comment:
Decided to add the constants here, so they are near the test case they're used.

Reply:
It's fine to keep them close, but let's keep them below the other code related to this test.

Reply:
Let's go with between `fn test_constants()` and `struct IfTemplate`.

Reply:
Unsure why that comment was first posted 3 hours after the other. I wrote them at the same time. However, both issues are now resolved 😅